server/src/main/java/com/google/codeu/servlets/LoginServlet.java

```diff
@@ -36,6 +47,11 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     // If the user is already logged in, redirect to their page
     if (userService.isUserLoggedIn()) {
       String user = userService.getCurrentUser().getEmail();
+      // stores the user in datastore if it's their first time signing in
+      if (datastore.getUser(user) == null) {
+        User newUser = new User(user, " ");
```

Comment:
Remove the space in the " " in `new User(user, " ");

Can you also write above this line `TODO(brianch): Refactor User to use builder pattern`

Thanks! Feel free to merge after this fix.